crates/fuel-core/src/database/database_description/off_chain.rs

```diff
@@ -11,6 +11,7 @@ impl DatabaseDescription for OffChain {
     type Column = fuel_core_graphql_api::storage::Column;
     type Height = BlockHeight;
 
+    // TODO[RC]: Do we bump this due to extended metadata?
```

Comment:
Maybe it will be right to increase it, but then you need to handle it here:

<img width="1058" alt="image" src="https://github.com/user-attachments/assets/985c4fa2-fbcb-4f75-87f3-1d7949c09001">

But maybe we don't need `version` function at all, if we version enum of the metadata.

Reply:
I removed the comment and decided to keep the `version()` at `0`, as it used to be, because I can't really feel the notion of this version number.

I'm good to update it, remove it or refactor if needed, but preferably in a follow-up PR.